app/assets/javascripts/training/components/training_slide_handler.jsx

```diff
@@ -76,11 +90,27 @@ const TrainingSlideHandler = () => {
     }));
   };
 
+
+  const [isShown, setIsShown] = useState(false);
+
+  // this function checks whether fastTrainingAlert should be shown or not.
+  const fast_Training_Alert_handler = () => {
```

Comment:
This function name should be in camelCase to match the style of the rest of our JS.